python/src/aac/plugins/check/check_aac_impl.py

```diff
@@ -69,12 +91,22 @@ def check_primitive_constraint(
     def check_schema_constraint(
         source_definition: Definition, check_me: Any, check_against
     ):
-        """Runs all the constraints for a given schema."""
+        """Helper method that runs all the constraints for a given schema.
+
+        Args:
+            source_definition (Definition): Source of the check_me field that we are evaluating
+            check_me (Any):                 The field being checked
+            check_against:                  The schema we are comparing the check_me field against
+
+        Returns:
+            (at check level) ExecutionResult: The result of the checks in this helper method
```

Comment:
This helper method has a LanguageError within, so need a raises entry as well explaining what causes the exception.

```suggestion
        Returns:
            (at check level) ExecutionResult: The result of the checks in this helper method
            
        Raises:
            LanguageError: When a definition for a field type cannot be found.
            LanguageError: When a field value is expected to be a list and is another type.
          
```